### PR TITLE
docs(samples): VAIS Answer - Add comment describing that all options are included by default

### PR DESCRIPTION
**Pull request by holtskinner**
## Description

Fixes b/369766709

### CODE REVIEW

File: discoveryengine/answer_query_sample.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 from google.cloud import discoveryengine_v1 as discoveryengine
 
 # TODO(developer): Uncomment these variables before running the sample.
+# NOTE: This sample shows all available answer phase options. Remove any options that you don't plan to use.
```
Comment (iennae) [inline discoveryengine/answer_query_sample.py:23]: nit: would recommend something a little more clear and specific for this comment. Suggestion: 

# The `answer_generation_spec` below includes all available answer phase options. 
# Modify or remove any options that are not relevant to your use case to optimize 
# the answer generation process.

**Review by iennae — APPROVED** [review]
I think the comment could be a little more clear and I gave an alternative, but it's up to you.

**Review by msampathkumar — APPROVED** [review]
Only docs updates. No code changes.

File: discoveryengine/answer_query_sample.py [hunk 2]
```diff
@@ -20,6 +20,10 @@
 from google.cloud import discoveryengine_v1 as discoveryengine
 
 # TODO(developer): Uncomment these variables before running the sample.
+# The `answer_generation_spec` below includes all available answer phase options.
```
Comment (msampathkumar) [inline discoveryengine/answer_query_sample.py:23]: Suggestion: Can you move the #TODO comment to be above project_id, location variables definition ?

**Pull request merged**